appveyor.yml

```diff
@@ -6,8 +6,6 @@ install:
 
 build_script:
   - cmd: fake.exe build target NuGet
-on_finish:
-  - ps: (New-Object 'System.Net.WebClient').UploadFile("https://ci.appveyor.com/api/testresults/nunit3/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\TestResult.xml))
```

Comment:
So we no longer publish test results?

Reply:
NUnit doesn't apparently produce TestResult.xml for dotnet test. It can be reintegrated if someone find a way to publish it.

Reply:
It will anyway fail the build and report in log in case of problem…